src/test/java/io/spring/start/site/StartApplicationHttpsTests.java

```diff
@@ -41,7 +41,9 @@
  * @author Stephane Nicoll
  */
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT, properties = "initializr.env.force-ssl=true")
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT, properties = {
+		"initializr.env.force-ssl=true",
+		"spring.config.location=file:./src/test/resources/application-test.yml" })
```

Comment:
This means that tests have to be ran from the module directory. This sounds kind of obvious but I don't like the idea of hardcode relative paths here.

Reply:
Where else would they be run from? We can use `classpath:...` I suppose. Want me to change it?